server/requirements.txt

```diff
@@ -1 +1 @@
-git+git://github.com/superdesk/newsroom-core.git@develop#egg=Newsroom-Core
+git+git://github.com/pablopunk/newsroom-core.git@SDAN-697#egg=Newsroom-Core
```

Comment:
I'll change this once https://github.com/superdesk/newsroom-core/pull/3 is merged